app/components/patientcard/patientcard.js

```diff
@@ -203,7 +203,7 @@ var PatientCard = React.createClass({
       <div>
         <div className="ModalOverlay-content">{"Are you sure you want to leave this person's Care Team? You will no longer be able to view their data."}</div>
         <div className="ModalOverlay-controls">
-          <button className="PatientInfo-button PatientInfo-button--secondary" type="button" onClick={this.overlayClickHandler}>Cancel</button>
+          <button className="PatientInfo-button PatientInfo-button--secondary" type="button" onClick={this.modalDismissHandler}>Cancel</button>
```

Comment:
This breaks, because the `overlayClickHandler` method below hasn't been renamed to `modalDismissHandler` (it should be).